internal/datastore/prepared_statements.go

```diff
@@ -262,6 +263,31 @@ const getDiscussionsForAutoPostString = `
 		FROM discussions
 		WHERE auto_post = true`
 
+const getDiscussionBySlugString = `
+		SELECT d.id,
```

Comment:
For the future: We should probably turn this select statement into a fragment. I know it's going to bite us if we don't lol.

Reply:
1000%